### ISSUE TITLE
Investigate zenhub

### ISSUE THREAD

**Issue by behrtam** [exercism/todo#178]
Just read about https://www.zenhub.io (free for OSS) and thought maybe this could help exercism somewhere. So this is more a bookmark or reminder to checkout zenhub and what features could help us when I find the time.

**Comment by kotp** [comment]
I also checked it out a couple days ago.  To me, it doesn't make sense to require a browser extension for those that want to contribute, it adds a layer, and is limited in what browsers you use to get the 'benefits'.  There will be some that would contribute but (I think) won't see the things that we are referring to, and will likely cause a bit more confusion in a somewhat complicated project.

The project itself looks very decent though.  For exercism though my vote is :no_entry: .

**Comment by kytrinyx** [comment]
I tried waffle.io a while back, and the extra layer thing wasn't working for me. I actually think that the thing we're missing is a human component: someone who is really great at helping keep on top of issues, triage, prioritization, roadmap, etc. There might be some good tooling that would help once a human helped define the big picture, but I'd want to see the human piece solved first.

**Comment by kotp** [comment]
Yeah, waffle.io was also ending up a no-go in my opinion as well.  Some of the same concerns.

**Comment by dogweather** [comment]
>  it doesn't make sense to require a browser extension for those that want to contribute

It wouldn't be required. One can use the issues as normal, blissfully unaware of the ZenHub enhancement.

**Comment by kytrinyx** [comment]
Yeah, it's still on my list to investigate this.
